Makefile

```diff
@@ -20,7 +20,7 @@ SCHEMA_FILE     := provider/cmd/pulumi-resource-kubernetes/schema.json
 GOPATH			:= $(shell go env GOPATH)
 
 JAVA_GEN 		 := pulumi-java-gen
-JAVA_GEN_VERSION := v0.5.2
+JAVA_GEN_VERSION := v0.8.0
```

Comment:
Note: PR is currently based on #2337. This change will not be in this PR once #2337 is merged to master and this PR is rebased.